core/odin/parser/parser.odin

```diff
@@ -416,7 +416,16 @@ expect_closing_brace_of_field_list :: proc(p: ^Parser) -> tokenizer.Token {
 		str := tokenizer.token_to_string(token)
 		error(p, end_of_line_pos(p, p.prev_tok), "expected a comma, got %s", str)
 	}
-	return expect_token(p, .Close_Brace)
+	expect_brace := expect_token(p, .Close_Brace)
+
+	if expect_brace.kind != .Close_Brace {
+		for p.curr_tok.kind != .Close_Brace && p.curr_tok.kind != .EOF {
```

Comment:
I'd probably also check for a semicolon (and make sure it is not a newline too).

Reply:
Alright added that. This should hopefully be good enough for now. I might later down the line try to recover the other fields, but the most important thing is to recover all the other declarations outside of the struct.